scripts/linters/pylint_extensions.py

```diff
@@ -1464,6 +1464,14 @@ class DocstringChecker(checkers.BaseChecker):
         )
     }
 
+    # Docstring section headers split up into arguments, returns, yields and
+    # and raises sections signifying that we are currently parsing the
+    # corresponding section of that docstring.
+    "DOCSTRING_SECTION_ARGS": "args",
```

Comment:
Did you test this manually?

Reply:
What do you mean by test manually?

Reply:
By running the linter after your changes.

Reply:
Done